.github/workflows/ci.yml

```diff
@@ -884,7 +884,7 @@ jobs:
         shell: bash
         run: |
           cd .setup/ansible
-          ansible-playbook --private-key /home/runner/.ssh/id_rsa -e 'ansible_user=runner' -i inventory/submitty playbooks/submitty_install.yml
+          ansible-playbook --private-key /home/runner/.ssh/id_rsa -e "ansible_user=runner submitty_install_submitty_version=${{ github.sha }}" -i inventory/submitty playbooks/submitty_install.yml
```

Comment:
When thinking about this more, would it be better to use the SHA of the commit, or the branch name, so you don't have to specify the refspec also.

Reply:
@lavalleeale I have the same question.

Reply:
I think it is important to include the refspec because investigating how this would affect external PRs suggests that it is the only way to ensure that the PR would be found by git. There is no downside to specifying the refspec since Ansible will still clone normally first.

Reply:
This is a way to do it, however it using a sha as a refspec is unusual. 

Do we care about PRs on a fork? This fix only works with PRs that are in the main Submitty repository. The only thing I can think that would be annoying is when working with my college to do some Submitty development, we had a fork of Submitty that we made PRs towards, and used the CI of that as to not clog the main Submitty repo (albeit this was back when tests took over an hour to run).

Reply:
PRs from branches that are not in the [Submitty/Submitty](https://github.com/Submitty/Submitty) will not allow switching to their branch. This seems to be the only way to allow people not associated with the submitty organization to have their PRs pass tests unless we add more ansible arguments to specify seperate refspecs and versions.